web/packages/teleterm/src/ui/services/clusters/clustersService.ts

```diff
@@ -149,7 +149,7 @@ export class ClustersService extends ImmutableStore<types.ClustersServiceState>
    * syncRootCluster is useful in situations where we want to sync the cluster _and_ propagate any
    * errors up.
    */
-  private async syncRootCluster(clusterUri: uri.RootClusterUri) {
```

Comment:
This used to be private just to signal that this method is used only in this file. Now that we need to sync the cluster details after creating the CMC role, I had to make it public again.

It used to be public when we had that sync button over the resource table.